asm2vec/tensors.py

```diff
@@ -0,0 +1,69 @@
+import os
+import torch
+import logging
+import asm2vec
+from asm2vec import utils
+from pathlib import Path
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+
+def calc_tensors(asm_path, tensor_path, model_path, epochs, device='cpu', lr=0.02) -> list:
```

Comment:
Can you include types for your args?
```suggestion
def calc_tensors(asm_path: str, tensor_path: str, model_path: str, epochs: int, device: str = 'cpu', learning_rate: float = 0.02) -> list:
```
And `lr` is not a good arg name, too unclear. Can we update to something like `learning_rate`? Verbose is better with naming